src/main/java/io/vertx/core/net/KeyManagerOptions.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2011-2020 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.net;
+
+import javax.net.ssl.KeyManager;
+
+/**
+ * @author Hakan Altindag
+ */
+class KeyManagerOptions extends KeyManagerFactoryOptions {
```

Comment:
why is this class necessary ?

Reply:
It is not necessary, I wanted to decouple the new KeyOptions by having two implementations, 1. KeyManagerOptions and 2. KeyManagerFactoryOptions as one can be build from the keymanagerfactory and the other one from just only a keymanager. But if you prefer to have a single implementation I can move the constructor to KeyManagerFactoryOptions